src/test/shared/codelens/sampleGoSamProgram.ts

```diff
@@ -0,0 +1,145 @@
+/*!
+ * Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as vscode from 'vscode'
+
+/**
+ * This script emits file contents from the stock SAM CLI app for Go, as well as
+ * the corresponding DocumentSymbols that would be generated by the Go language server.
+ *
+ */
+
+export function getDocumentSymbols(): vscode.DocumentSymbol[] {
+    const symbols: vscode.DocumentSymbol[] = []
+
+    const badFuncSymbol: vscode.DocumentSymbol = new vscode.DocumentSymbol(
+        'notAHandler',
```

Comment:
I feel like the answer is probably no, but out of my ignorance for the VSC api, is there no nicer way to do this?

Reply:
The "nicer" way to do it is to implement a barebones CodeLens provider.